src/install/unix.rst

```diff
@@ -230,9 +234,55 @@ CouchDB no longer ships with any daemonization scripts.
 The couchdb team recommends `runit <http://smarden.org/runit/>`_ to
 run CouchDB persistently and reliably. Configuration of runit is
 straightforward; if you have questions, reach out to the CouchDB
-user mailing list.
+user mailing list or IRC-channel #couchdb in FreeNode network.
+
+Let's consider configuring of runit on Ubuntu 16.04. The following
+steps should be considered only as an example. Some details can vary
```

Comment:
Details will vary by operating system and distribution. Check your system's package management tools for specifics.